api/src/backend/api/rls.py

```diff
@@ -87,7 +87,9 @@ def create_sql(self, model: Any, schema_editor: Any) -> Any:
                 f"{grant_queries}{self.grant_sql_query.format(statement=statement)}"
             )
 
-        full_create_sql_query = f"{self.rls_sql_query}{policy_queries}{grant_queries}"
+        full_create_sql_query = (
+            f"{self.rls_sql_query}" f"{policy_queries}" f"{grant_queries}"
+        )
```

Comment:
I don't understand this change, can you explain it?

Reply:
I think this is some IDE auto formatting. I havev't intentionally changed it. Reverting this.